recipe/meta.yaml

```diff
@@ -11,20 +11,22 @@ source:
 
 build:
   number: 0
-  script: '{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv '
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv
   noarch: python
 
 requirements:
   host:
     - pip
-    - python >=3.5
+    - python
     - setuptools >=44
     - setuptools_scm >=3.4.3
+    - toml
```

Comment:
pip includes a vendored `tomli` library, the `toml` dependency may not be necessary

Reply:
toml mentioned in the upstream pyproject.toml https://github.com/alexmojaki/stack_data/blob/80ea27810df097dd844351266983d3516425901f/pyproject.toml#L2